test/sagemaker_tests/tensorflow/tensorflow1_training/integration/sagemaker/test_horovod.py

```diff
@@ -22,6 +24,9 @@
 RESOURCE_PATH = os.path.join(os.path.dirname(__file__), '..', '..', 'resources')
 
 
+@pytest.mark.integration("horovod")
+@pytest.mark.model("mnist")
+@pytest.mark.multinode("multinode(2)")
```

Comment:
nit: `pytest.mark.multinode()` seems to have inconsistent arguments provided to it. It is `"multinode"` in some places, and `"multinode(<num_nodes>)"` in others.

Can this be made consistent?

Reply:
In places where it's not explicit how many nodes are being run, I left as multinode